keyboards/keebit/coral/info.json

```diff
@@ -0,0 +1,152 @@
+{
+    "manufacturer": "keeb.it!",
+    "keyboard_name": "coral",
+    "maintainer": "numToStr",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "lto": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "rows": ["GP29", "GP28", "GP14", "GP6", "GP7"],
+        "cols": ["GP1", "GP0", "GP27", "GP26", "GP15", "GP5"]
+    },
+    "processor": "RP2040",
+    "url": "https://github.com/keeb-it/coral",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED",
+        "force_nkro": true,
+        "polling_interval": 1,
+        "suspend_wakeup_delay": 500
```

Comment:
Is this needed?

Reply:
- i added `suspend_wakeup_delay` to try to fix wakeup issue on windows but still didn't fix the issue. I'll removed it.
- `force_nkro` idk why i added that, i'll remove it as well.
- don't we need `vid`? It was added by `qmk new-keyboard`

Reply:
I've now removed force_nkro and suspend_wakeup_delay.